common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -128,7 +128,7 @@ protected[core] case class BlackBoxExec(override val image: String, code: Option
 }
 
 protected[core] case class SequenceExec(components: Vector[FullyQualifiedEntityName]) extends Exec(Exec.SEQUENCE) {
-    override def size = components.map(c => c.size).reduce(_ + _)
+    override def size = if (components.size > 0) components.map(c => c.size).reduce(_ + _) else (0 B)
```

Comment:
Or do we even need size calculation on Sequences? Isn't the "size" of the list bound by its count anyway? Could we always return 0 byte?